docs/reference/field-ref.asciidoc

```diff
@@ -2,32 +2,45 @@
 [role="xpack"]
 == Elastic Security ECS field reference
 
-This section lists ECS fields Elastic Security uses to display data.
+This section lists ECS fields used by {es-sec} to provide an optimal SIEM and/or security analytics experience to users.  These fields are used to display data, provide rule previews, enable detection by prebuilt detection rules, provide context during rule triage and investigation, escalate to cases, and more.
```

Comment:
Sometimes {es-sec} fails to render correctly for some reason, and {elastic-sec} has been more reliable. Also, IMO "and/or" doesn't fit into our typical conversational tone -- just "and" seems fine here.
```suggestion
This section lists ECS fields used by {elastic-sec} to provide an optimal SIEM and security analytics experience to users.  These fields are used to display data, provide rule previews, enable detection by prebuilt detection rules, provide context during rule triage and investigation, escalate to cases, and more.
```